login.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { 
+  AppRegistry, 
+  Text,
+  StyleSheet,
+  View,
+  TextInput,
+  Button,
+  Alert,
+  Image,
+} from 'react-native';
+import { Actions, Router, Scene } from 'react-native-router-flux'
+
+import Home from './Home';
```

Comment:
Doesn't seem like you are using Home in this file at all. So this isn't necessary